src/utils/rewriteLink.ts

```diff
@@ -102,6 +101,22 @@ const rewriteLink = (
                 domToReact(children, parseOptions)
             )
         }
+        const hashMatches = href.match(/(\.md#([A-Za-z0-9_-]+))$/);
```

Comment:
I think you'll need to escape the final `-`
```suggestion
        const hashMatches = href.match(/(\.md#([A-Za-z0-9_\-]+))$/);
```

Reply:
I don't believe you do? I mean, it's working in the preview: https://5efaa115098fad000928d5d0--ss-userhelp.netlify.app/en/4/optional_features/content_blocks/

You can see the raw source here: https://github.com/dnadesign/silverstripe-elemental/blame/4/docs/en/userguide/index.md#L6 where the `.md` is in the link and removed on the rendered docs site.

Reply:
ok all good then